pimd/pim_cmd_common.c

```diff
@@ -1079,6 +1080,9 @@ void ip_pim_ssm_show_group_range(struct pim_instance *pim, struct vty *vty,
 		vty_json(vty, json);
 	} else
 		vty_out(vty, "SSM group range : %s\n", range_str);
+#else
+	/* TBD with PIM6_SSM_STANDARD_RANGE */
```

Comment:
You can define FF3x::/96 as PIM_SSM_STANDARD_RANGE for PIMv6.
#if PIM_IPV == 4
#define PIM_SSM_STANDARD_RANGE "232.0.0.0/8"
#else
#define PIM_SSM_STANDARD_RANGE "FF3x::/96"

Reply:
Fixed, Thankyou